### PR TITLE
feat(cost-analysis): add show-favorite-only in cost-analysis LNB

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/cost-explorer/CostExplorerLNB.vue [hunk 1]
```diff
@@ -101,19 +102,33 @@ const state = reactive({
             },
             favoriteType: FAVORITE_TYPE.COST_ANALYSIS,
         }));
+
+        const currenctQueryMenuListFilterdByFavoriteState = filterFavoriteItems(currentQueryMenuList);
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/CostExplorerLNB.vue:106]: quibble: too long!!!😂

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/cost-explorer/CostExplorerLNB.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/cost-explorer/CostExplorerLNB.vue:106]: nitpick: typo.. `currenct`

Comment (skdud4659) [reply apps/web/src/services/cost-explorer/CostExplorerLNB.vue:106]: lol😂

**Review by skdud4659 — APPROVED** [review]
LGTM~!

**Review by WANZARGEN — APPROVED** [review]
LGTM

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Comment by piggggggggy** [comment]
Thanks! I apply your reviews and refactor them!

**Pull request merged**